src/main/java/edu/greenblitz/robotName/subsystems/arm/EndEffector/RollerUtils/RollerConstants.java

```diff
@@ -4,6 +4,7 @@ public class RollerConstants {
 
 	public static final double ROLL_FORWARD_POWER = 1;
 	public static final double ROLL_BACKWARD_POWER = -1;
+	public static final double TIME_AFTER_OBJECT_IN_ROBOT = 0.5;
```

Comment:
```suggestion
	public static final double TIME_UNTIL_NOTE_EXIT_ARM= 0.5;
```
not sure if i understood, but if i did change the name to this if not to a different one